include/zarr.types.h

```diff
@@ -0,0 +1,133 @@
+#ifndef H_ACQUIRE_ZARR_TYPES_V0
+#define H_ACQUIRE_ZARR_TYPES_V0
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    typedef enum
+    {
+        ZarrStatus_Success = 0,
```

Comment:
In some instances, you assign a default value of zero, while in others, you don't. Is there a specific reason for this inconsistency?

Reply:
These are all collected from different places, I just moved them and didn't really look at it. ZarrVersion_2 defaults to 2 to eliminate any surprises.